sites/docs/src/pages/styles.module.css

```diff
@@ -216,9 +216,12 @@
 
 
 .vitessceClear {
-  text-align: center;
+  text-align: right;
   width: 100%;
-  margin-bottom: 10px;
+  padding: 10px;
+  display: 'flex';
+  flex: 1;
+  margin-top: auto;
```

Comment:
so that the "Edit" button appears on the top right side of the screen, side by side with the Demo description